code/datums/world_topic.dm

```diff
@@ -168,9 +168,16 @@
 	.["security_level"] = get_security_level()
 	.["round_duration"] = SSticker ? round((world.time-SSticker.round_start_time)/10) : 0
 	// Amount of world's ticks in seconds, useful for calculating round duration
+	
+	//Time dilation stats.
+	.["time_dilation_current"] = SStime_track ? SStime_track.time_dilation_current : 0
+	.["time_dilation_avg"] = SStime_track ? SStime_track.time_dilation_avg : 0
+	.["time_dilation_avg_slow"] = SStime_track ? SStime_track.time_dilation_avg_slow : 0
+	.["time_dilation_avg_fast"] = SStime_track ? SStime_track.time_dilation_avg_fast : 0

```

Comment:
Are the SS checks necessary? Can Topics even be received before world/New()?

Reply:
world/New() is basically the last step of initialization, so yes.

Reply:
@Cyberboss asked the wrong question, these things can happen before world/New(), yes, but subsystems are created at global var init time, and no topic can come in before the first sleep in global var inits, and nothing in global vars initialization before the mc should be sleeping since the mc is one of the first real globals.